C/CUTEst/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder
 
 name = "CUTEst"
-version = v"0.2.0"
+version = v"0.2.3"
```

Comment:
Why is the version changing if the source is the same?

Reply:
I was confused the first time I submitted this and thought `version` was the version of the recipe. Shall I leave it as it was?

Reply:
Well, ideally the version should be that of the upstream package

Reply:
Duh. I wanted to write 2.0.3. Let me fix it.